src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/ConfirmPanelCompositionHandler.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.sscs.callback.handlers;
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.callback.CallbackHandler;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.DispatchPriority;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.service.PanelCompositionService;
+
+@Slf4j
+@Service
+public class ConfirmPanelCompositionHandler implements CallbackHandler<SscsCaseData> {
+
+    private PanelCompositionService panelCompositionService;
+
+    @Autowired
+    public ConfirmPanelCompositionHandler(PanelCompositionService panelCompositionService) {
+        this.panelCompositionService = panelCompositionService;
+    }
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+
+        return callbackType.equals(CallbackType.SUBMITTED)
+            && callback.getEvent() == EventType.CONFIRM_PANEL_COMPOSITION
+            && callback.getCaseDetails().getCaseData().getAppeal() != null
```

Comment:
You could just add '&& callback.getEvent() == EventType.UPDATE_OTHER_PARTY_DATA' here and do away with your other handler? And maybe rename the handler to one that is generic? Thats kind of how we do most of the other submitted handlers in evidence share

Reply:
I though it would be better keeping it separate in case we need to do specific things to each event in future, TBF Rasika will be doing that already, as the notifications will be triggered just for UploadOtherParty event.

Reply:
as I commented on ccd-definition pr, can tribunal trigger the CONFIRM_PANEL_COMPOSITION event from UPDATE_OTHER_PARTY_DATA submitted handler? and then handle it only this handler. Because UPDATE_OTHER_PARTY_DATA event should trigger OTHER_PARTY_ADDED event as well.

Reply:
I think this is a discussion to be held with @jackmaloney  as I did first in tribunals and then was asked to redo it in evidence-share...